### PR TITLE
apply WhoopsListener for error handler listener

### PR DESCRIPTION
**Pull request by samsonasik**
it uses https://github.com/pennyphp/excp-handler . To make it testable, @gianarb please activate packagist hook https://github.com/pennyphp/excp-handler/issues/5 , When it active, composer.lock ready for re-update again and ready for check.

### CODE REVIEW

File: config/di.php [hunk 1]
```diff
@@ -1,14 +1,22 @@
...
-        $eventManager->attach("dispatch_error", [$container->get(\App\EventListener\DispatcherExceptionListener::class), "onError"]);
-        $eventManager->attach("*_error", [$container->get(\App\EventListener\ExceptionListener::class), "onError"]);
+        $eventManager->attach("dispatch_error", [$container->get(WhoopsListener::class), "onError"]);
+        $eventManager->attach("*_error", [$container->get(WhoopsListener::class), "onError"]);
+        
+        // this is usage if we want to use templated error handler
+//       \App\EventListener\ExceptionListener::class => \DI\object(\App\EventListener\ExceptionListener::class)
+//                ->constructor(\DI\get("template")),
+//       \App\EventListener\DispatcherExceptionListener::class => \DI\object(\App\EventListener\DispatcherExceptionListener::class)
+//               ->constructor(\DI\get("template")),
```
Comment (samsonasik) [inline config/di.php:13]: commented so user can use templated error handler when they want.

**Comment by samsonasik** [comment]
travis error because of the `penny/excp-handler` package in packagist didn't updated. The hook need to be activated, and after that, the composer.lock can be re-updated again. /cc @gianarb

**Comment by gianarb** [comment]
Are you that it is a problem of packagist?
https://travis-ci.org/pennyphp/penny-skeleton-app/jobs/96010691

**Comment by samsonasik** [comment]
yes, it read composer.lock which is still old lib, i will update composer.lock in a few

**Comment by samsonasik** [comment]
@gianarb composer.lock and test updated, should be green now ;)

**Comment by samsonasik** [comment]
@gianarb travis green now \m/

**Comment by samsonasik** [comment]
tested in screenshot below:

tested

**Comment by gianarb** [comment]
Thanks!!

**Pull request merged**